fem/bilininteg_diffusion_pa.cpp

```diff
@@ -1601,12 +1574,8 @@ static void SmemPADiffusionApply3D(const int NE,
          {
             MFEM_FOREACH_THREAD(qx,x,Q1D)
             {
-               const int i = qi(qx,dy,Q1D);
-               const int j = dj(qx,dy,D1D);
-               const int k = qk(qx,dy,Q1D);
-               const int l = dl(qx,dy,D1D);
-               B[i][j] = b(qx,dy);
-               G[k][l] = g(qx,dy) * sign(qx,dy);
+               B[qx][dy] = b(qx,dy);
+               G[qx][dy] = g(qx,dy);
```

Comment:
I think it's better to remove the `x` and `y`:
```suggestion
               B[q][d] = b(q,d);
               G[q][d] = g(q,d);
```

Reply:
`d` is already used as local variable name and it is a way to be consistent with the `MFEM_FOREACH_THREAD` arguments.